lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/HeartBeatManager.cs

```diff
@@ -84,32 +87,20 @@ public HeartBeatManager(EvaluatorSettings settings, IRemoteIdentifier remoteId)
             }
         }
 
-        [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1307:AccessibleFieldsMustBeginWithUpperCaseLetter", Justification = "Intended to be private, exposed now before using future injection")]
-        public EvaluatorRuntime _evaluatorRuntime { get; set; }
+        public EvaluatorRuntime Evaluatorruntime { get; set; }
 
         [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1307:AccessibleFieldsMustBeginWithUpperCaseLetter", Justification = "Intended to be private, exposed now before using future injection")]
-        public ContextManager _contextManager { get; set; }
+        public ContextManager Contextmanager { get; set; }
 
-        public EvaluatorSettings EvaluatorSettings
-        {
-            get
-            {
-                return _evaluatorSettings;
-            }
-
-            private set
-            {
-                _evaluatorSettings = value;
-            }
-        }
+        public EvaluatorSettings EvaluatorSettings { get; private set; }
 
         public void Send(EvaluatorHeartbeatProto evaluatorHeartbeatProto)
         {
             lock (_queuedHeartbeats)
             {
-                if (_evaluatorSettings.OperationState == EvaluatorOperationState.RECOVERY)
+                if (EvaluatorSettings.OperationState == EvaluatorOperationState.RECOVERY)
```

Comment:
No, it is to reference to a property. I have changed to use _evaluatorSettings read only variable to avoid confusion.